### DISCUSSION TITLE
Change Language

### DISCUSSION THREAD

**Discussion by walikor** [Leon280698/CT#17] [Q&A] [answered]
hi, I have the steam version and I wanted to change the game to Italian, I went to the System.ini file and changed the Language line to ita and to Italian, but the game remains in English, instead with the basic version of steam the language changes , is there another way?

**Comment by leon-bckl** [comment, marked as answer]
The steam version only has the English language files

**Reply by walikor** [reply]
ok, strange because in the shop the Italian is mentioned....(https://store.steampowered.com/app/6000/STAR_WARS_Republic_Commando/) and the app have the italian file (https://steamdb.info/depot/6005/)
 I have to try in case with the gog version.

**Reply by walikor** [reply]
ok, sorry for the double post, but I solved it, yes the steam version has only the language in english, while the gog version is multi language, the only thing to be careful is that if you start the game with the ct.exe file the game is in English, instead with the original exe the game becomes in Italian, with your fantastic fix :)
